tests/modules/asset_groups/resources/test_patch_asset_group.py

```diff
@@ -212,25 +212,19 @@ def test_patch_asset_group_sighting_as_sighting(
     encounter_guids = [e['guid'] for e in response.json['encounters']]
 
     # Set first encounter sex to male
-    response = utils.patch_via_flask(
+    response = asset_group_utils.patch_asset_group_sighting_as_sighting(
```

Comment:
Using the AGS util rather than the underlying one. 
I'm not sure it's clear to everyone but the path bit can be whatever you want in the UI. 
I hit an issue with this as it was looking for guid and in the sighting response it seems to be id that is the field used. 
:warning: Really not sure about this bit. Should guid also be there as the same value?